config.docker-template.php

```diff
@@ -27,6 +27,18 @@
 } else if (!empty($port)) {
     $CFG->wwwroot .= ":{$port}";
 }
+
+if (($themedevel = getenv('MOODLE_DOCKER_THEMEDEVEL')) !== false) {
```

Comment:
It's a bad practice to use two commands in one line.

Reply:
How would you write it?
```
$themedevel = getenv('MOODLE_DOCKER_THEMEDEVEL');
if ($themedevel !== false) {
```

?